docs/faq.rst

```diff
@@ -0,0 +1,37 @@
+Frequently Asked Questions
+===============================
+
+.. glossary::
+
+	Is the information or data I add open access?
+		No, nothing is open access on CERN Analysis Preservation (except for the projects own source code). It is designed to be a save environment for CERN physicists to use from the very beginning of starting their analysis and at any given moment in its lifetime. It is also meant to increase this lifetime. Access to your analysis is always restricted to your collaboration and the permissions you assign.
```

Comment:
I would move the last sentence "Access to your analysis..." more to the front. Crucial bit of information :)